starters/gatsby-starter/src/templates/index.js

```diff
@@ -41,6 +48,14 @@ export default function Home({ ...props }) {
     >
       <HomepageHeader />
       <section>
+        <div className="mt-4 flex flex-row justify-center items-center">
```

Comment:
UI nit: Could you move this **under** the PostGrid? 

It's confusing when this is the first thing you see on the page, more so when it's without the previous and next buttons
![image](https://github.com/pantheon-systems/pantheon-content-cloud-sdk/assets/87580113/d3b66df2-1696-4786-960f-3c2118ab346a)